packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -798,6 +797,17 @@ export default {
             );
         },
 
+        /**
+         * Scroll to the first inline error, no matter which one it is.
+         */
+        scrollToFirstInlineError () {
+            this.$nextTick(() => {
+                const firstInlineError = document.querySelector('[data-js-error-message]');
+
+                this.scrollToElement(firstInlineError, { offset: -100 });
```

Comment:
Probably would be cleared up by adding a comment to the `scrollToElement` function, but it's not obvious what -100 refers to (is it -100 pixels? Why 100 specifically – is that the height of an error element?)